src/Core/Framework/App/Privileges/Privileges.php

```diff
@@ -0,0 +1,245 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\App\Privileges;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use Psr\EventDispatcher\EventDispatcherInterface;
```

Comment:
usually we are typ-hinting `Symfony\Contracts\EventDispatcher\EventDispatcherInterface` :thinking: 

probably we are using this on other places, too. we should make it consistent with a PHPStan rule at some day mabe :thinking: